src/engraving/libmscore/bracket.cpp

```diff
@@ -144,10 +144,10 @@ void Bracket::setStaffSpan(int a, int b)
         if (score()->styleSt(Sid::MusicalSymbolFont) == "Leland") {
             v = qMin(4, v);
         }
-        // total default height of a system of n staves / height of a 5 line staff
-        qreal dist = score()->enableVerticalSpread() ? score()->styleS(Sid::maxAkkoladeDistance).val() : score()->styleS(
-            Sid::akkoladeDistance).val();
-        _magx = v + ((v - 1) * dist / 4.0);
+
+        // 1.125 is a "magic" number based on akkoladeDistance/4.0 (default value 6.5).
+        _magx = v + ((v - 1) * 1.125);
```

Comment:
Any special reason not to take `score()->styleS(Sid::akkoladeDistance).val()` here then?

Reply:
Yes, using this parameter is exactly what causes the issue this PR tries to solve :wink: 
Also, this parameter is a parameter used when vertical adjustment of staves is disabled, using it when this adjustment is **enabled** is very confusing (especially since the parameter is disabled).
The (new) idea of this PR is the make the scaling of the brace independent of the spacing, only the number of staves spanned is used. See also [this comment](https://github.com/musescore/MuseScore/pull/9903#issuecomment-986057266).